include/ignition/gui/qml/Chart.qml

```diff
@@ -0,0 +1,277 @@
+                anchors.rightMargin: parent.width * 0.05
+                Text { anchors.centerIn: parent; text: "x"; color: "white"}
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        exitAnimation.start();
+                        // unSubscribe from the transport
+                        main.unSubscribe(main.chartID, component.topic, component.path);
+                        // delete the series points and deattache it from the chart
+                        chart.deleteSeries(component.topic + "-"+ component.path)
+                        // delete the field info component
+                        component.destroy();
+                    }
+                }
+                NumberAnimation {
+                    id: enterAnimation
+                    target: exitBtn; property: "opacity"; duration: 200
+                    easing.type: Easing.InOutQuad; from: 0; to: 1;
+                }
+                NumberAnimation {
+                    id: exitAnimation
+                    target: exitBtn; property: "opacity"; duration: 200;
+                    easing.type: Easing.InOutQuad; from: 0.85; to: 0;
+                }
+            }
+        }
+    }
+
+    // ================== Chart ============================
+    ChartView {
+        id : chart
+        width: parent.width
+        height: parent.height * 0.9
+
+        // animations
+        antialiasing: true
+        opacity: 1
+        backgroundRoundness: 10
+        animationDuration: 400
+        animationOptions: ChartView.SeriesAnimations
+
+        theme: (Material.theme == Material.Light) ? ChartView.ChartThemeLight: ChartView.ChartThemeDark
+
+        property var serieses: ({})
+        property var colors: ["red","blue","cyan","yellow","green","lightGray"]
+        property int indexColor: 0
+
+        // for vertical line hover
+        MouseArea {
+            id:areaView
+            anchors.fill:parent
+            hoverEnabled: true
+            property bool flag: false
+            cursorShape: (multiChartsMode) ? Qt.PointingHandCursor : Qt.ArrowCursor
+
+            onEntered: {
+                if(flag && ! multiChartsMode)
+                {
+                    // TODO : draw a vertical line and show its point value
+                    flag=false;
+                }
+                else if (multiChartsMode)
+                {
+                    chart.opacity = 0.7;
+                }
+            }
+            onExited: {
+                if (multiChartsMode)
+                {
+                    chart.opacity = 1;
+                }
+            }
+
+            onClicked: main.clicked(chartID);
+        }
+
+        // initial animation
+        Component.onCompleted:{
+            initialAnimation.start();
+        }
+        NumberAnimation {
+            id: initialAnimation
+            running: false
+            target: view
```

Comment:
When I add a new chart I get this warning:

`[GUI] [Wrn] [Application.cc:649] [QT] qrc:/qml/Chart.qml:218: ReferenceError: view is not defined`